### ISSUE TITLE
Failed to reproduce paper results

### ISSUE THREAD

**Issue by BrennoCaldato** [zzangjinsun/NLSPN_ECCV20#20]
Hi @zzangjinsun. Thanks for sharing the code.

I tried to reproduce the paper results with Kitti and NYU datasets, using the pre-trained weights you provided. 

I followed the Readme instructions but the output depth images are not accurate. Here are the quantitative results I'm getting for the NYU dataset:

> Metric   |  RMSE: 0.0922  MAE: 0.0348  iRMSE: 0.0139  iMAE: 0.0050  REL: 0.0117  D^1: 0.9955  D^2: 0.9993  D^3: 0.9999

Did you face this issue before?

**Comment by zzangjinsun** [comment]
Hi @BrennoCaldato ,
I think you got the correct results, but D^1, D^2, and D^3 are not in percent so you can multiply 100 to compare with the results in the paper.
